<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publisherAddingForm</class>
 <widget class="QWidget" name="publisherAddingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add publisher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string>Publisher name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Creation date</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="addBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
